### DISCUSSION TITLE
DolphinScheduler里面如何在一个工作流设定2个任务串行依赖关系，使得任务A先跑，同时不管成功失败，任务B会接着跑

### DISCUSSION THREAD

**Discussion by Sheldon-xs** [apache/dolphinscheduler#16511] [Q&A]
还请各位技术大拿指点下：
DolphinScheduler里面如何在一个工作流设定2个任务串行依赖关系，使得任务A先跑，同时不管成功失败，任务B会接着跑

**Comment by reele** [comment]
串个switch，成功失败都指到B

**Reply by Sheldon-xs** [reply]
感谢回复，可否提供个简单例子

**Comment by llllkid** [comment]
在A和B中间加个`CONDITIONS`节点，节点里这么配置：
![image](https://github.com/user-attachments/assets/32b31bdb-7231-4f60-be83-6cce6c32f65f)

**Reply by llllkid** [reply]
这么配完了，无论成功还是失败B都会执行

**Reply by Sheldon-xs** [reply]
明白了，谢谢
